projects/ccf-organ-info/src/app/core/services/organ-lookup/organ-lookup.service.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable } from '@angular/core';
+import { AggregateResult, Filter, SpatialEntity, SpatialSceneNode } from 'ccf-database';
+import { ALL_ORGANS, OrganInfo } from 'ccf-shared';
+import { Observable, of } from 'rxjs';
+import { map } from 'rxjs/operators';
+
+import { DataSourceService } from '../data-source/data-source.service';
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class OrganLookupService {
+  private readonly organs = ALL_ORGANS;
+
+  constructor(private readonly source: DataSourceService) { }
+
+  getOrganInfo(iri: string, side?: OrganInfo['side'], _sex: Filter['sex'] = 'Female'): Observable<OrganInfo | undefined> {
+    let info = this.organs.find(item => item.id === iri);
```

Comment:
I added different logic for this, please review my latest commit and add that in here.